system-parachains/collectives/collectives-polkadot/src/ambassador/migration.rs

```diff
@@ -0,0 +1,388 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Migrations.
+
+use frame_support::{pallet_prelude::*, traits::OnRuntimeUpgrade, weights::Weight};
+
+pub(crate) mod add_accounts {
+	use super::*;
+	#[cfg(feature = "try-runtime")]
+	use alloc::vec::Vec;
+	use frame_support::parameter_types;
+	use pallet_ranked_collective::{
+		Config, IdToIndex, IndexToId, MemberCount, MemberRecord, Members, Rank,
+	};
+
+	parameter_types! {
+		// Public key (hex)
+		pub const Addresses: [(Rank, [u8; 32]); 125] = [
```

Comment:
One comment please on how this list was created?  
I see its in a gdoc but would be nice to have a link here as well.